src/pages/baan-profile.tsx

```diff
@@ -0,0 +1,235 @@
+import Image from 'next/image';
+
+const profilePic = '/images/pfp-placeholder.svg';
+
+const BaanProfile = () => {
+    const bgStyle: string = 'lg:mb-none lg:mb-6 p-auto py-8 px-8';
+    const bottonStyle: string =
+        'lg:mx-4 lg:px-6 text-sm py-2 mx-auto px-3 rounded-lg ring-4 ring-gray-300';
+    interface GroupMember {
+        name: string;
+        surname: string;
+        imgUrl: string;
+        role: string;
+    }
+    interface SelectedBaan {
```

Comment:
We already have the `IBaan` interface, or if you prefer, the `IShortBaan` interface, available in `src/types/baan.ts`. Please use that instead.